src/components/OptionsList/BaseOptionsList.js

```diff
@@ -77,6 +80,53 @@ class BaseOptionsList extends Component {
         Log.hmmm('[OptionsList] scrollToIndex failed', info);
     }
 
+    /**
+     *
+     * @param {Array} data – this is a flat array of all the sections with all the section headers inline. It comes from react-native in this format
+     * @param {Number} index - the index of the current item in the `data` array
+     * @returns {Object}
+     */
+    getItemLayout(data, index) {
+        const optionHeight = this.props.optionMode === CONST.OPTION_MODE.COMPACT ? variables.optionRowHeightCompact : variables.optionRowHeight;
+        const sectionHeaderHeight = variables.optionsListSectionHeaderHeight;
+
+        let offset = 0;
+        let currentIndex = 0;
+        let isItemSectionHeader = false;
+        _.any(this.props.sections, (section) => {
```

Comment:
```suggestion
        _.some(this.props.sections, (section) => {
```
NAB but more common to see `some` vs. `any`. Had to look up `any` would not have looked up `some`.